src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,26 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void mainApp(String[] args) {
 
+        Machine[] machines = new Machine[3];
+        machines[0] = new Excavator();
+        machines[1] = new Bulldozer();
+        machines[2] = new Truck();
```

Comment:
better init like this:
```suggestion
        Machine[] machines = new Machine[] {
        new Excavator(),
        new Bulldozer(),
        new Truck()
        };
```